### ISSUE TITLE
Mysterious command "failed with error: Error: spawn g++11.2 ENOENT"

### ISSUE THREAD

**Issue by hellige** [microsoft/vscode-cmake-tools#3063]
I mentioned this in follow-up to a previous issue, but never got any reply, so I suspect no one noticed it. This is still happening, exactly as described below, with version 1.14.16.

One strange behavior remains. With 1.13.7, my output from above becomes:
```
[proc] Executing command: /home/mhellige/drw/test/../cpp-build-tools/gcc11.2.0-1 -v
... [ everything is fine, cmake runs ok, etc. then finally: ]
[proc] The command: gcc11.2 -v failed with error: Error: spawn gcc11.2 ENOENT
[proc] The command: g++11.2 -v failed with error: Error: spawn g++11.2 ENOENT
```

(And the stray directories are no longer created in my home dir.)

I still can't figure out those last two `ENOENT` lines... Where in the world is `gcc11.2` coming from? My compiler binary is called `gcc11.2.0-1`, as you can see above. I can't find that string in any file in the workspace, nor in `~/.vscode` or `~/.local/share/CMakeTools`. I've tried enabling trace logging and everything else I can think of, but I can't for the life of me figure out where it's getting the string `gcc11.2`...

I'm not sure if this is a different bug, or a configuration issue, or what, but I would be very grateful for any insight!

_Originally posted by @hellige in https://github.com/microsoft/vscode-cmake-tools/issues/2737#issuecomment-1252728317_

**Comment by haozia816** [comment]
都2025年9月了，还是相同错误

**Comment by hellige** [comment]
Indeed. Seems like MS only works on stupid AI stuff these days. Maybe they can get Copilot to fix this issue.

**Comment by lygstate** [comment]
Mark it, I also face the same issue, let's digging what's happened.
